src/main/java/org/shanerx/tradeshop/itrade/IShopCreateEventListener.java

```diff
@@ -124,7 +125,8 @@ public void onSignChange(SignChangeEvent event) {
         }
 
         if (chest != null) {
-            setName((InventoryHolder) chest.getState(), "o:" + plugin.getSettings().getString("itrade-shop-name"));
+            changeInvName(chest.getState(), ((InventoryHolder) chest.getState()).getInventory().getName().toString(),
+                    Collections.singletonList(plugin.getServer().getOfflinePlayer(plugin.getSettings().getString("itrade-shop-name"))), Collections.emptyList());
```

Comment:
Wouldn’t `Arrays.asList(T...t)` would suffice? Much shorter than `Collections.singletonList(...)`

Reply:
for single item lists the collections way is more efficient as it doesn't actually create an array in memory